gateway/src/main/scala/filodb/gateway/GatewayServer.scala

```diff
@@ -79,14 +79,16 @@ object GatewayServer extends StrictLogging {
     val sourceConfigPath = trailArg[String](descr = "Path to source config, eg conf/timeseries-dev-source.conf")
     val genHistData = toggle(noshort = true, descrYes = "Generate prom-histogram-schema test data and exit")
     val genDeltaHistData = toggle(noshort = true, descrYes = "Generate delta-histogram-schema test data and exit")
-    val genPromData = toggle(noshort = true, descrYes = "Generate Prometheus-schema test data and exit")
+    val genGaugeData = toggle(noshort = true, descrYes = "Generate Prometheus gauge-schema test data and exit")
+    val numMetrics = opt[Int](short = 'm', default = Some(1), descr = "# of metrics - use 2 to test binary joins")
+    val publishIntervalSecs = opt[Int](short = 'i', default = Some(10), descr = "Publish interval between samples")
     verify()
   }
 
   //scalastyle:off method.length
   def main(args: Array[String]): Unit = {
     val userOpts = new GatewayOptions(args)
-    val numSamples = userOpts.samplesPerSeries() * userOpts.numSeries()
+    val numSamples = userOpts.samplesPerSeries() * userOpts.numSeries() * userOpts.numMetrics()
```

Comment:
Would be clearer if `numSeries` were `numSeriesPerMetric`

Reply:
done